### ISSUE TITLE
NPE in com.qulice.maven.DependenciesValidator

### ISSUE THREAD

**Issue by yegor256** [teamed/qulice#720]
I'm getting this, in version 0.16:

```
[INFO] --- qulice-maven-plugin:0.16:check (jcabi-check) @ java-sdk ---
[INFO] Starting Checkstyle validator
[INFO] LICENSE found: file:/Volumes/ssd2/code/sttc/java-sdk/LICENSE.txt
log4j:WARN No appenders could be found for logger (com.puppycrawl.tools.checkstyle.ConfigurationLoader).
log4j:WARN Please initialize the log4j system properly.
log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.
[INFO] No Checkstyle violations found in 42 files
[INFO] Finishing Checkstyle validator
[INFO] Starting PMD validator
[INFO] No PMD violations found in 35 files
[INFO] Finishing PMD validator
[INFO] Starting XML validator
[INFO] /src/site/site.xml: skipped
[INFO] Finishing XML validator
[INFO] Starting CodeNarc validator
[INFO] CodeNarc not required since no groovy files in /Volumes/ssd2/code/sttc/java-sdk/src
[INFO] Finishing CodeNarc validator
[INFO] Starting FindBugs validator
[INFO] Finishing FindBugs validator
[INFO] Calling org.apache.maven.plugins:maven-enforcer-plugin:1.0-beta-1:enforce...
[INFO] Calling com.ning.maven.plugins:maven-duplicate-finder-plugin:1.0.7:check...
[INFO] Checking compile classpath
[INFO] Checking runtime classpath
[INFO] Calling org.codehaus.mojo:cobertura-maven-plugin:2.5.1:check...
[INFO] Cannot perform check, instrumentation not performed - skipping.
[INFO] This is not an SVN project
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 21.823 s
[INFO] Finished at: 2016-02-18T20:10:42-08:00
[INFO] Final Memory: 51M/937M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.qulice:qulice-maven-plugin:0.16:check (jcabi-check) on project java-sdk: Execution jcabi-check of goal com.qulice:qulice-maven-plugin:0.16:check failed. NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.qulice:qulice-maven-plugin:0.16:check (jcabi-check) on project java-sdk: Execution jcabi-check of goal com.qulice:qulice-maven-plugin:0.16:check failed.
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:212)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution jcabi-check of goal com.qulice:qulice-maven-plugin:0.16:check failed.
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:145)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	... 20 more
Caused by: java.lang.NullPointerException
	at java.lang.String.startsWith(String.java:1393)
	at java.lang.String.startsWith(String.java:1422)
	at com.qulice.maven.DependenciesValidator$ExcludePredicate.apply(DependenciesValidator.java:186)
	at com.qulice.maven.DependenciesValidator$ExcludePredicate.apply(DependenciesValidator.java:167)
	at com.google.common.base.Predicates$NotPredicate.apply(Predicates.java:335)
	at com.google.common.collect.Iterators.indexOf(Iterators.java:796)
	at com.google.common.collect.Iterators.any(Iterators.java:706)
	at com.google.common.collect.Iterables.any(Iterables.java:621)
	at com.google.common.collect.Collections2$FilteredCollection.isEmpty(Collections2.java:184)
	at com.qulice.maven.DependenciesValidator.validate(DependenciesValidator.java:78)
	at com.qulice.maven.CheckMojo.run(CheckMojo.java:88)
	at com.qulice.maven.CheckMojo.doExecute(CheckMojo.java:59)
	at com.qulice.maven.AbstractQuliceMojo.execute(AbstractQuliceMojo.java:176)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	... 21 more
[ERROR]
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

Don't know how to reproduce :(

**Comment by yegor256** [comment]
@krzyk actually, I know how to reproduce. just check out this branch: https://github.com/sttc/java-sdk/pull/18 it doesn't build because of this bug

**Comment by krzyk** [comment]
@yegor256 it was fixed in 0.16.1 (see #676)

**Comment by yegor256** [comment]
@krzyk sorry, my fault. thanks
